projects/plugins/jetpack/json-endpoints/class.wpcom-json-api-list-posts-v1-1-endpoint.php

```diff
@@ -220,6 +221,10 @@ public function callback( $path = '', $blog_id = 0 ) {
 			$query['has_password'] = false;
 		}
 
+		if ( isset( $args['include'] ) ) {
+			$query['post__in'] = $args['include'];
```

Comment:
Do we need to do any kind of sanitizing / casting here, to ensure that we'll have the proper data depending on what was passed (array or int)?

Reply:
That's a good idea! [Fixed it here](https://github.com/Automattic/jetpack/pull/29149/commits/22278727539e13044e1ae3dc7cdc327fa1906060).